lightning/src/ln/channel.rs

```diff
@@ -1672,6 +1672,54 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		cmp::min(self.value_to_self_msat as i64 - self.get_outbound_pending_htlc_stats().1 as i64, 0) as u64)
 	}
 
+	fn commit_tx_fee_msat(&self, num_htlcs: u64) -> u64 {
```

Comment:
Why is `usize` preferred here?

Reply:
Since `num_htlcs` corresponds to a vector length, it would be better to keep it as its original type until in needs to be converted to something of a specific size (e.g., inside `commit_tx_fee_msat`).

There could be an argument for converting it early if `num_htlcs` was something serializable on the wire or had a limited range. But if just needed in intermediary calculations, then letting the function do the conversion may be more appropriate since the caller shouldn't have to worry about this detail.